code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -445,3 +445,47 @@
 
 /datum/species/golem/bananium/get_spans()
 	return list(SPAN_CLOWN)
+
+
+/datum/species/golem/runic
+	name = "Runic Golem"
+	id = "runic"
+	limbs_id = "cultgolem"
+	sexes = FALSE
+	info_text = "As a <span class='danger'>Runic Golem</span>, you possess eldritch powers granted by the Elder God Nar'Sie."
+	species_traits = list(NOBREATH,RESISTHOT,RESISTCOLD,RESISTPRESSURE,NOFIRE,NOGUNS,NOBLOOD,RADIMMUNE,VIRUSIMMUNE,PIERCEIMMUNE,NODISMEMBER) //no mutcolors
+
+	var/obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/golem/phase_shift

```

Comment:
that's odd, when I first previewd the diff they didn't have the identation to match the typepath, did you adjust the spacing or did I just fail to use my eyes correctly?